platform/qt/src/qmapboxgl.cpp

```diff
@@ -1775,6 +1775,10 @@ QMapboxGLPrivate::QMapboxGLPrivate(QMapboxGL *q, const QMapboxGLSettings &settin
            .withConstrainMode(static_cast<mbgl::ConstrainMode>(settings.constrainMode()))
            .withViewportMode(static_cast<mbgl::ViewportMode>(settings.viewportMode()));
 
+    if (!m_localFontFamily.isEmpty()) {
+        options.withLocalFontFamily(m_localFontFamily.toStdString());
```

Comment:
If think you can just override without checking if `isEmpty()`.

Reply:
This was here because I was assuming `PingFang` as the default for MapOptions, but as you mentioned below - it is not. This simplifies this code to the proposed changed.